examples/variables.tf

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Required Paramaters
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to use, set via environment variable TF_VAR_ibmcloud_api_key"
+  type        = string
+  sensitive   = true
+}
+
+##############################################################################
+# Module Level Variables
+##############################################################################
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default = "au-syd"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default = "test-landing-zone-vpc"
+}
+
+variable "vpc_name" {
```

Comment:
no need for `var.vpc_name` and `var.prefix` in the example. Let just pass the `var.prefix` value for `vpc_name` and remove `var.vpc_name` from the example.